admin-dev/themes/new-theme/js/product-page/attributes.js

```diff
@@ -52,4 +52,20 @@ export default function() {
         $(`.js-attribute-checkbox[data-value="${e.attrs.value}"]`).prop('checked', false);
       }
     });
+
+  $('.form-control').each(function () {
+    var attr = null;
+    if (this.hasAttribute('counter') === true) {
+      attr = $(this).attr('counter');
+      if (typeof attr === typeof undefined || attr === false) {
```

Comment:
Why not an each on `.form-control input[counter=*]` or something like that ? because, some page have so mush form-control... ? WDYT